gradle.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# These are global settings for the gradle build process.
+
+org.gradle.jvmargs=-Xmx5g -XX:MaxMetaspaceSize=2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.caching=true
+org.gradle.console=rich
+
```

Comment:
Github won't let me add a comment on the empty file `gradle/wrapper/gradle-wrapper.jar`, so tagging this in the wrong place... 

Should this be a .gitignore?

Reply:
It isn't an empty file, it's just a jar package so GitHub presumably can't show a textual preview. It's necessary, as it contains targets of the gradlew shell script.